public/message.ts

```diff
@@ -0,0 +1,63 @@
+import { updateScroll } from "./ui";
+import { gameOver } from "./game";
+import { getLogsElement, getElementById } from "./helpers";
+
+// TODO: What type to use for message - string or MessageEvent?
+export const handleMessage = (message: MessageEvent): void => {
+  // string type introduces errors
+  let msg = JSON.parse(message.data);
+  console.log(msg);
+  let currentText = getLogsElement().innerHTML;
```

Comment:
* ~probably doesn't need to be a let (could be const)~
* move it into the case statements that actually use it (define your variables as close as possible to where they are used.